src/pages/about.js

```diff
@@ -71,52 +71,47 @@ function About() {
                 <p className="story">
                   At Vets Who Code, we take a small cohort of veterans and spouses twice a year and
-                  over the course of sixteen weeks train them in programming with a deep focus on
-                  Javascript, the language of the web. We do this all remotely using the best tools
-                  and resources the tech community has to offer. We do this at zero cost to the
-                  troop beyond them contributing to one of our teams and helping each other become
-                  better programmers.
+                  train them in programming with a deep focus on JavaScript, over the course of sixteen weeks. 
+                  We do this all remotely using the best tools and resources the tech community has to offer. 
+                  In addition to contributing and helping each other become better programmers, we do this at zero cost.
                 </p>
 
                 <h2>How We Do This</h2>
                 <p className="story">
-                  How we accomplish this mission is through a process of <i>Crawl, Walk, Run</i>
-                  where as we teach them programming we build upon each lesson in deeper dives so
-                  that they become better programmers through each iteration. All this while being
-                  lead by instructors who are also veterans, and programmers, and alumni of the
-                  program. That way there are no excuses, the person teaching you the craft has been
+                  How we accomplish this mission is through a process of <i>Crawl, Walk, Run </i>
+                  where we build upon each lesson in deeper dives so that veterans become better programmers through each iteration. 
+                  Instructors leading the program are also veterans, programmers, and alumni of the
+                  program. There are no excuses, the instrcutor has been
                   through every success and failure you will experience and has ultimately succeeded
                   on the path of becoming a paid programmer.
                 </p>
 
                 <h2>WE NEED YOUR HELP</h2>
                 <p className="story-last">
                   As our country is going through this unprecedented time with COVID-19, people are
-                  wanting to learn the skills of today and tomorrow to future-proof themselves so
-                  that they can provide for their families. Furthermore due to the nature of
-                  for-profit code schools many are just now learning how to do the work remotely,
-                  which we have been doing for years. As a result we are having an unprecedented
+                  wanting to learn the skills of today and tomorrow to future-proof themselves in order to provide for their families. 
+                  Furthermore, due to the nature of for-profit coding schools, many are now learning how to do the work remotely,
```

Comment:
Remove "Furthermore" and start the sentence with "Due"

Reply:
Change "many are now learning how to do the work remotely," to "many are now learning how to work remotely,"